platform/src/main/java/org/hillview/utils/TestTables.java

```diff
@@ -18,6 +18,7 @@
 
 package org.hillview.utils;
 
+import com.sun.tools.javac.util.Assert;
```

Comment:
The issue was that the junit assert wasn't in the `main` scope, only in the `test` scope. I've added it as a dependency.